### PR TITLE
Add link to alteration page in the mutation effect description

### PR DESCRIPTION
**Pull request by zhx828**
when the allele is unknown but multiple alternative alleles at the same position have mutation effects

This fixes https://github.com/oncokb/oncokb/issues/3552

### CODE REVIEW

File: src/main/webapp/app/pages/annotationPage/MutationEffectDescription.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { AlterationPageLink } from 'app/shared/utils/UrlUtils';
+import WithSeparator from 'react-with-separator';
+import SummaryWithRefs from 'app/oncokb-frontend-commons/src/components/SummaryWithRefs';
+
+const MutationEffectDescription: React.FunctionComponent<{
+  hugoSymbol: string;
+  description: string | undefined;
+}> = props => {
+  // Add link to alteration page when the allele is unknown but multiple alternative alleles at the same position have mutation effects
```
Comment (calvinlu3) [inline src/main/webapp/app/pages/annotationPage/MutationEffectDescription.tsx:10]: Do we want to also add a link to alteration page when there is 1 AA?

Comment (zhx828) [reply src/main/webapp/app/pages/annotationPage/MutationEffectDescription.tsx:10]: There would not be a case like that. When only one AA, we include the AA mutation description in the `description`

File: src/main/webapp/app/pages/annotationPage/MutationEffectDescription.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  const description = props.description || '';
+  const addlMeSeparator = 'however, we have mutation effect descriptions for';
```
[the remark this answers is not in the document]
Comment (zhx828) [reply src/main/webapp/app/pages/annotationPage/MutationEffectDescription.tsx:12]: `additionalMutationEffectSeparator` does it make sense? I can spell it out actually if it does.

[the remark this answers is not in the document]
Comment (calvinlu3) [reply src/main/webapp/app/pages/annotationPage/MutationEffectDescription.tsx:12]: That makes sense. I was just not familiar with the abbreviations for additional. I understood that we were referring to the ME.
```
From google:
The main ways to abbreviate the word additional are: Add'l. Addt'l. Add.
```

**Review by calvinlu3 — APPROVED** [review]
Looks good

**Pull request merged**